roles.js

```diff
@@ -85,10 +23,10 @@ const fullGameCategoriesThatAreActuallyILs = [
     "9d8pgl6k"
 ];
 
-const srcDataFile = "./src_data.json";
-
-exports.init = (c) => {
+exports.init = (c, s, l) => {
     client = c;
+    sql = s;
+    log = l;
```

Comment:
Maybe just name the parameters `client`, `sql`, and `log` so you don't have to do this reassignment?

Reply:
changed this